app/component/route/route.scss

```diff
@@ -33,7 +34,8 @@ $route-header-favourite-icon-selected-hover-fill-color: $background-color;
   overflow: hidden;
   color: $gray;
   font-size: $font-size-small;
-  margin-left: 1em;
+  margin-left: 0px;
```

Comment:
I don't know if we have a CSS style guide but I think it says use "0" instead of "0px" ;-)